graphql_compiler/compiler/ir_lowering_match.py

```diff
@@ -9,17 +9,17 @@
 to simplify the final code generation step.
 """
 
-from collections import namedtuple
+from collections import deque, namedtuple
```

Comment:
Can you split out these changes into a separate PR, since they are unrelated to the documentation?